modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1911,6 +1911,11 @@ return view.extend({
 				]).then(this.renderContents.bind(this));
 			};
 
+			if (bss.ssid == null) {
```

Comment:
I think you also need to actually handle the new `ssid` option somehow around [line 1791](https://github.com/openwrt/luci/pull/4565/files#diff-adf712ab47ca2e5c052d0ad45de5737b2be6860525cd51ec8f424c7e3a43b78aR1791)

Reply:
Didn't notice that.. Tought the data were saved automagically.